dist/tokens/ts/dimensions.js

```diff
@@ -0,0 +1,8 @@
+/**
+ * Do not edit directly
+ * Generated on Tue, 21 Nov 2023 18:48:18 GMT
+ */
+
+module.exports = {
+  "contentAreaMax": "1196px"
```

Comment:
just adding this content area for now, started adding breakpoints but became more complicated - and didn't think it was a priority 

any else to add here?

Reply:
would it make sense to use a "sidebar/nav" prefix here? I'm assuming this value wouldn't apply to the top-nav layout in patient app?

Reply:
yeah, makes sense to me 👍